src/components/layouts/app-layout/app-layout.styles.ts

```diff
@@ -14,8 +16,13 @@ export const styles = createStyles({
     flex: "1 1 auto"
   },
 
+  hasDrawer: {
+    marginLeft: drawerWidth
+  },
+
   drawer: {
-    width: 420
+    width: drawerWidth,
+    boxShadow: "0 0 30px rgba(0,0,0,0.2)"
```

Comment:
```suggestion
    boxShadow: "0 0 30px rgba(0, 0, 0, 0.2)"
```
Spaces improve readability :)